### DISCUSSION TITLE
Error on Google Play Store Publish For "Changes cannot be sent for review automatically"

### DISCUSSION THREAD

**Discussion by ZetiMente** [codemagic-ci-cd/codemagic-docs#1131] [Q&A] [answered]
Hi, 

Error on Google Play Store Publish For "Changes cannot be sent for review automatically". This option does not work on CodeMagic. 

<img width="772" alt="Screen Shot 2022-11-04 at 2 37 22 PM" src="https://user-images.githubusercontent.com/76985631/200069636-78e9e5be-4099-4737-bac0-62cdf211d20a.png">
<img width="398" alt="Screen Shot 2022-11-04 at 2 37 46 PM" src="https://user-images.githubusercontent.com/76985631/200069704-cb33304c-9062-4a60-9e52-a43f56109e99.png">

**Comment by himesh-cm** [comment, marked as answer]
Hi @ZetiMente, please check this [link](https://docs.codemagic.io/troubleshooting/common-google-play-errors/#changesnotsentforreview-errors)
